docs/general/userscripts.md

```diff
@@ -13,19 +14,37 @@ AdGuard can significantly broaden website functionality by acting as a userscrip
 
 These userscripts come directly from the AdGuard developers, and we can guarantee that they are effective and safe. Remember: whenever you download a userscript from an unknown source, you expose yourself to risk, as some scripts may be harmful to your computer. Get new scripts only from the sources you trust 100%. Furthermore, here are [some examples](#more-userscripts) of popular sites with scripts.
 
-- **AdGuard Extra**. An extension that blocks advertising in difficult cases when the usual filter-based approach is not enough.
+#### AdGuard Extra
```

Comment:
I'd suggest using increasing headers size overall on this page and then you can use `###` here